<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QValueDateInputDlg</class>
 <widget class="QWidget" name="QValueDateInputDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="m_touchradioBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>触摸</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_BitradioBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_PouupckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>61</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>启用弹出键盘：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_allowinputbitlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>102</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>允许输入位地址：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Monitor_Address_EntryBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_AllowInputBitlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>102</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LB0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_keyspBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>56</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="m_KeygroupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>键盘坐标</string>
   </property>
   <widget class="QSpinBox" name="m_xKeyspinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X坐标：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y坐标：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_yKeyspinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="m_AutoChangeBitckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动复位</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
